Problem solving/assignment.py

```diff
@@ -0,0 +1,59 @@
+'''
+a1,a2,a3,a4,a5 ............an
+Algorithm:
+1.Alice chooses first 	
+2.So if the sum of alice is even then its bob turn
+3.As the sum is even bob needs to remove all even numbers so that alice remains with odd numbers
+4.Bob chooses an even number
+5.Alice scans the array again if there is an even number then alice picks it up
+else alice picks up an odd number
+6.if sum of alice is odd then bob picks up an odd number 
+7.Then this process keeps on going till there are no elements left
+8.If alice sum is even then alice wins or bob wins
+
+'''
+
+inputs = [1,2,10,3,15]
+Alice_score = 0
+while(len(inputs)!=0):
+    for i in inputs:
+        if (i%2 == 0 and Alice_score%2 == 0):
+            Alice_score += i
+            inputs.remove(i)
+            print("Alice turn ",i)
+
+            break
+        elif(i%2 != 0 and Alice_score%2 != 0):
+            Alice_score += i
+            inputs.remove(i)
+            print("Alice turn ",i)
+
+            break
+        else:
```

Comment:
The if-elif-else block is having same logic for all the conditions. How exactly are the conditions different?

Consider the following testcase: [1,2,10,3,15]
The problem statement says that both the players play optimally. According to your code, in the first iteration Alice chooses 1. But since Alice plays optimally, she would choose either 2 or 10 as those are even numbers.